### ISSUE TITLE
Module de téléchargement de fichiers

### ISSUE THREAD

**Issue by remijean** [remijean/ZwiiCMS#156]
Module permettant au utilisateurs de télécharger des fichiers depuis le site, avec classement dans catégories.
Les modérateurs/admin pourront ajouter des fichiers.

**Comment by simon511000** [comment]
Ce module serait parfait pour faire un espace numérique de travail, exemple, un professeur créer une catégorie : chapitre 1, et dedans, il met tous les cours relatifs au chapitre 1 👍

**Comment by remijean** [comment]
@simon511000 Yes et lorsque j'aurais terminé le nouveau gestionnaire de fichiers ça devrait être plutôt simple à implémenter 😄 !

**Comment by simon511000** [comment]
Cette fonctionnalité sortira dans la version 9 de Zwii ?

**Comment by remijean** [comment]
Non dans l'une des prochaines mises à jour de la 8. J'ai juste pris un peu de retard à cause du boulot 😞 .
